src/database/Connector.java

```diff
@@ -124,16 +124,44 @@ public boolean verifyAccount(String username, String password) {
             String passwordInDB = resultPassword.getString(1);
             // password does not match
             if (!passwordInDB.equals(password)) {
-              System.err.println("the password does not match");
+              //System.err.println("the password does not match");
               return false;
             }
-            System.out.println("the account exists");
+            //System.out.println("the account exists");
             return true;
           }
         }
       }
     } catch (SQLException e) {e.printStackTrace();return false;}
-    System.err.println("the account does not exist");
+    //System.err.println("the account does not exist");
+    return false;
+  }
+
+  /**
+   * verify if the account username input by users matches an account in the
+   * database
+   *
+   * @param username
+   * @return true if the account exists, false if the account does not exist
+   */
+  public synchronized boolean verifyAccountExists(String username) {
+
+    String sqlSelectPassword = "select password from " + tableName + " where username = ?";
```

Comment:
I think the method has been updated since last commit. This is in my code:
`public boolean addAccount(String username, String password);`
use usrname to get the password in the database, and then compare to the password passed in as a parameter to validate the account. In the future, the password will be encrypted, and I am currently looking into digesting the password